ytools/y2tool/rubocop_yast_style.yml

```diff
@@ -39,3 +43,11 @@ Style/NegatedIf:
 # use "raise" instead of "fail"
 Style/SignalException:
   EnforcedStyle: only_raise
+
+# do not force %r
+Style/RegexpLiteral:
+  Enabled: false
+
+# Forbid and/or usage in conditionals
+Style/AndOr:
```

Comment:
no, it should be forbidded there.
so you can do `a = b[:c] or raise "invalid data in #{b}"` but cannot do `if a and b`